packages/live-share/src/internals/RoleVerifier.ts

```diff
@@ -83,9 +83,19 @@ export class RoleVerifier implements IRoleVerifier {
         return this._getRequestCache.cacheRequest(clientId, () => {
             return waitForResult(
                 async () => {
-                    const rolesResult = await this._host.getClientRoles(
-                        clientId
-                    );
+                    let rolesResult;
```

Comment:
might be better to declare the type here

Reply:
adding as what's expected but it will dynamically check types later in the code.